docs/source/getting_started/install.rst

```diff
@@ -3,29 +3,30 @@
 *********************
 Installation with PIP
 *********************
-Once you've installed Ansys 2021R1 or newer, you can install DPF with:
+Once Ansys 2021 R1 or later is installed, you can install the 
```

Comment:
Could we please have a way specify here that it's from 2021 R1 to the latest version of Ansys

Reply:
@cbellot000 It already says 2021 R1 or later--that is standard Ansys phrasing....

Reply:
Sorry, my bad